pocket_lab/pl_gui_iio_wrapper.c

```diff
@@ -274,7 +274,7 @@ int32_t pl_gui_get_chn_attr_avail_options(const char *attr_name,
 		char *attr_val, uint32_t chn_indx, uint32_t dev_indx)
 {
 	struct iio_device *iio_dev;
-	struct iio_ch_info chn_info;
+	struct iio_ch_info chn_info = { 0 };
```

Comment:
Is channel info is not applicable here as it is get channel attribute method?

Reply:
I think I missed to update chn_info later. I just need channel number here. Will update this